MuonAnalysis/MomentumScaleCalibration/test/Macros/RooFit/MultiHistoOverlapAll_Base.C

```diff
@@ -220,15 +235,15 @@ void splitOptionRecursive(string rawoption, vector<string>& splitoptions, char d
 }
 
 void getCustomRanges(TString type, TString resonance, int iP, float minmax_plot[2]);
-void MultiHistoOverlapAll_Base_one(string files, string labels, string colors, string linestyles, string markerstyles, TString directory, TString resonance, TString type, bool switchONfit);
+void MultiHistoOverlapAll_Base_one(string files, string labels, string colors, string linestyles, string markerstyles, TString directory, TString resonance, TString type, bool switchONfit, bool AutoSetRange, float CustomMinY, float CustomMaxY);
```

Comment:
Dear Andres,

I assigned a default value in the function definition, so I cannot assign it also in the function declaration. I could assign the default value in the function declaration and remove it from the function definition though, is that what you were suggesting?

Thanks in advance,
Andrea

Reply:
> I could assign the default value in the function declaration and remove it from the function definition though, is that what you were suggesting?

Not exactly. But that's a good idea.

Also, please make sure your commit offers an explanation of the change you're applying and make sure it is unique. Right now I see you have three different commits with the same description. It is better to avoid 'Merge' commits, check `git rebase`

Reply:
Thank you very much for the tip, I'll implement the change.
As for the commits I'll try to be more careful next time when preparing a pull request, thanks for the information.